src/OpenApi/gen/XmlComments/MemberKey.cs

```diff
@@ -14,14 +14,19 @@ internal sealed record MemberKey(
     MemberType MemberKind,
     string? Name,
     string? ReturnType,
-    string[]? Parameters) : IEquatable<MemberKey>
+    List<string>? Parameters) : IEquatable<MemberKey>
 {
-    private static readonly SymbolDisplayFormat _typeKeyFormat = new(
+    private static readonly SymbolDisplayFormat _withTypeParametersFormat = new(
         globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.Included,
         typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces,
         genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters);
 
-    public static MemberKey FromMethodSymbol(IMethodSymbol method, Compilation compilation)
+    private static readonly SymbolDisplayFormat _sansTypeParametersFormat = new(
```

Comment:
Maybe call it `without` rather than `sans`?

Reply:
Updated.